tests/validators/array.test.ts

```diff
@@ -177,6 +177,20 @@ describe('ArrayValidator', () => {
 		});
 	});
 
+	describe('Unique', () => {
+		const uniquePredicate = predicate.unique;
+
+		test('GIVEN a non-unique array THEN throws CombinedPropertyError', () => {
```

Comment:
This isn't right... This should not throw, but an array of `['Hello', 'Hello']` should